src/auto/configure

```diff
@@ -4554,6 +4554,15 @@ $as_echo "yes" >&6; };;
 $as_echo "no" >&6; };;
 esac
 
+{ $as_echo "$as_me:${as_lineno-$LINENO}: checking for RHEL/Fedora" >&5
+$as_echo_n "checking for RHEL/Fedora... " >&6; }
+RH_OS=`cat /etc/os-release | grep -e 'Fedora\|Red Hat'`
```

Comment:
RH_OS=`cat /etc/os-release 2>/dev/null | grep -e 'Fedora\|Red Hat'`

and below

Reply:
@mattn I'll change that, thank you for the review!